app/views/DR/HomeScreen/index.js

```diff
@@ -164,6 +165,46 @@ class HomeScreen extends Component {
       state: { confirmed, deaths, recovered, current, refreshing, notified },
     } = this;
 
+    const dashboardCards = () => {
```

Comment:
I think it would be cleaner and better to read if you move this piece of code to other file